src/Simulation/Common/SimulatorBase.cs

```diff
@@ -165,6 +173,25 @@ public void DisableLogToConsole()
             OnLog -= Console.WriteLine;
         }
 
+        /// <summary>
+        ///     Adds an event to the OnStackTrace event that logs stack traces
+        ///     as plain text via the OnLog event (e.g.: for console output).
+        /// </summary>
+        public void EnableDefaultStackTraceHandling()
```

Comment:
Maybe name this `EnablePrintException` is not handling StackTrace but how exceptions are handled...